src/main/java/srijana/workshop_1/Task_3.java

```diff
@@ -0,0 +1,40 @@
+package srijana.workshop_1;
```

Comment:
Change already exists in PR https://github.com/tpolishchuk/java-core-trainings/pull/14/files, needs to be removed